src/Keboola/DbExtractor/Extractor/Extractor.php

```diff
@@ -90,9 +106,14 @@ public function createSshTunnel(array $dbConfig)
         $sshConfig['privateKey'] = isset($sshConfig['keys']['#private'])
             ?$sshConfig['keys']['#private']
             :$sshConfig['keys']['private'];
-        $tunnelParams = array_intersect_key($sshConfig, array_flip([
-            'user', 'sshHost', 'sshPort', 'localPort', 'remoteHost', 'remotePort', 'privateKey'
-        ]));
+        $tunnelParams = array_intersect_key(
```

Comment:
As before, this does not seem more readable IMO

Reply:
yeah, that's the cs rule though